src/systemtest/test_level2.py

```diff
@@ -1206,6 +1215,105 @@ def test_get_area_v5(
             url += '?%s' % urllib.parse.urlencode(param)
         self.validate_v5_results(url, nr_expected, expected_l2)
 
+    @pytest.mark.parametrize('min_scanid,expect_scanids', (
+        (0, range(7014791072, 7014791088)),
+        (7014791088, range(7014791088, 7014791102)),
+        (7014791101, [7014791101]),
+    ))
+    def test_get_area_v5_paging_returns_ok_data(
+            self, odinapi_service, lot_of_fake_data, min_scanid,
+            expect_scanids):
+        url = make_dev_url('{host}/rest_api/v5/level2/{project}/area'.format(
+            host=odinapi_service, project=PROJECT_NAME,
+        ))
+        param = dict(
+            start_time='2015-03-02', min_pressure=1, document_limit=1000,
+            min_scanid=min_scanid)
+        url += '?%s' % urllib.parse.urlencode(param)
+        r = requests.get(url)
+        products = r.json()["Data"]
+        assert(
+            set([product["ScanID"] for product in products])
+            == set(expect_scanids)
+        )
+
+    @pytest.mark.parametrize('min_scanid,expect_link,expect_url', (
+        (0, True, 'min_scanid=7014791088'),
+        (7014791088, False, None),
+    ))
+    def test_get_area_v5_paging_returns_ok_links(
```

Comment:
rename test to clarify it also tests for when theres no link

Reply:
or better make two tests

Reply:
renamed to _ok_or_no_links_